manifests/letsencrypt-issuer.yaml

```diff
@@ -0,0 +1,62 @@
+{{ if eq .dns.type "route53" }}
+---
+kind: Secret
+apiVersion: v1
+metadata:
+  name: prod-route53-credentials
+type: Opaque
+stringData:
+  secret-access-key: "{{.dns.secretKey}}"
+{{ end }}
+---
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-prod
```

Comment:
This would need to instead of the `ingress-ca` issuer as that is the one used by default when signing ingress certs, so this probably needs to move into the cert-manager/install.go and create the issuer programmatically.